### ISSUE TITLE
add rel nofollow to submit feedback link.

### ISSUE THREAD

**Issue by alexskr** [ncbo/bioportal_web_ui#121]
bots/crawlers should not follow submit feedback link

**Comment by jvendetti** [comment]
@alexskr - was this issue addressed by changes to the robots.txt file (https://github.com/ncbo/bioportal_web_ui/blob/master/config/robots/production.txt#L11)? Or, did you want the rel=nofollow in addition to that?

**Comment by alexskr** [comment]
yes, I think we should also add rel=nofollow just to be on the safe side
